payments-core/src/test/java/com/stripe/android/payments/PaymentIntentFlowResultProcessorTest.kt

```diff
@@ -186,7 +185,129 @@ internal class PaymentIntentFlowResultProcessorTest {
 
             verify(
                 mockStripeRepository,
-                times(PaymentIntentFlowResultProcessor.MAX_RETRIES)
+                times(PaymentFlowResultProcessor.MAX_RETRIES)
+            ).refreshPaymentIntent(
+                eq(clientSecret),
+                eq(requestOptions)
+            )
+        }
+
+    @Test
+    fun `3ds2 canceled with processing intent should succeed`() =
```

Comment:
Spoke on slack.

The name here indicates that if we end up in the processing state we should emit a success but ... the test is actually checking to see if a processing intent eventually becomes a succeeded, then we can emit success. Maybe we could name this better or add a comment if the name ends being too long. Ditto for the others. 

I can look at the test and PR and figure out what's going on, but maybe renaming would make it more explicit. Up to you.